ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/ws/WebSocketStart.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.ws;
+
+import java.util.Dictionary;
+import java.util.Hashtable;
+import org.eclipse.jetty.http.pathmap.PathSpec;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.websocket.server.WebSocketUpgradeFilter;
+import org.eclipse.jetty.websocket.servlet.WebSocketCreator;
+import org.osgi.framework.BundleContext;
+import org.osgi.service.http.whiteboard.HttpWhiteboardConstants;
+
+public class WebSocketStart {
+
+  private WebSocketCreator webSocketCreator;
+
+  private BundleContext bundleContext;
+
+  public WebSocketStart(BundleContext bundleContext, WebSocketCreator webSocketCreator) {
+    this.webSocketCreator = webSocketCreator;
+    this.bundleContext = bundleContext;
+    try {
+      start();
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public void start() throws Exception {
+
+    ServletContextHandler servletContextHandler =
+        new ServletContextHandler(ServletContextHandler.SESSIONS);
+    servletContextHandler.setContextPath("/search/catalog/ws");
```

Comment:
❓ I'm wondering if these configurable?

Reply:
These probably should be constants at a minimum if not configurable

Reply:
Same as above